src/scss/tooltip.scss

```diff
@@ -1,4 +1,10 @@
 @import "o-overlay/main";
+@import "o-grid/main";
+
+@if $o-tooltip-is-silent == false {
```

Comment:
👍 Glad you chose not to include the whole grid. Could this be moved to `main.scss`?